secrets/rules/hardcodedPassword.go

```diff
@@ -0,0 +1,68 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/rules"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func HardcodedPassword() *config.Rule {
+	// This regex is the output regex of 'generic-api-key' rule from gitleaks, with the next changes:
+	// 1. gitleaks/gitleaks#1267
+	// 2. gitleaks/gitleaks#1265
+	// 3. Minimum length of 4 characters (was 10)
```

Comment:
This regex is the output regex of 'generic-api-key' rule from gitleaks, with the next changes:

1. gitleaks/gitleaks#1267
2. gitleaks/gitleaks#1265
3. Minimum length of 4 characters (was 10)

Reply:
💪